common/thorhelper/thorcommon.hpp

```diff
@@ -166,7 +166,7 @@ interface IExtRowStream: extends IRowStream
     virtual CRC32 queryCRC() const = 0;
 };
 
-interface IExtRowWriter: extends IRowWriter
+interface ILogicalRowWriter: extends IRowWriter
```

Comment:
not super sold on the 'Logical' part of the name of this interface.. (or ILogicalRowStream).

```
IExtRowStream : IRowStream
    Similar to ILogicalRowStream, but with different functions and filters set later.
    Only used in thor.  
    FUTURE: Should ideally merge functionality info ILogicalRowStream
```
Actually, where is ILogicalRowStream?

Reply:
I will rename to ISimpleRowStream and IRowStream in a later PR and similar for writing.